components/blocks/CompareBox.tsx

```diff
@@ -1,27 +1,29 @@
-import React, { useState, useEffect, useRef } from 'react'
-import { FaCircle } from 'react-icons/fa'
-import Slider from 'react-slick'
-import 'slick-carousel/slick/slick.css'
-import 'slick-carousel/slick/slick-theme.css'
-import css from 'styled-jsx/css'
-import Image from 'next/image'
+import React, { useState, useEffect, useRef } from 'react';
+import { FaCircle } from 'react-icons/fa';
+import Slider from 'react-slick';
+import 'slick-carousel/slick/slick.css';
+import 'slick-carousel/slick/slick-theme.css';
+import css from 'styled-jsx/css';
+import Image from 'next/image';
+import { tinaField } from 'tinacms/dist/react';
+import { IoMdInformationCircleOutline } from 'react-icons/io';
 
 function hexToRgba(hex, alpha) {
-  let r = parseInt(hex.slice(1, 3), 16)
-  let g = parseInt(hex.slice(3, 5), 16)
-  let b = parseInt(hex.slice(5, 7), 16)
-  return `rgba(${r}, ${g}, ${b}, ${alpha})`
+  let r = parseInt(hex.slice(1, 3), 16);
+  let g = parseInt(hex.slice(3, 5), 16);
+  let b = parseInt(hex.slice(5, 7), 16);
+  return `rgba(${r}, ${g}, ${b}, ${alpha})`;
 }
```

Comment:
I would leave a comment on why you've done it this way (or if it can be fixed later, a tech debt PBI).

Reply:
done